examples/auth/app/templates/login.hbs

```diff
@@ -1,11 +1,11 @@
 <h1>Login</h1>
 
-<form onsubmit={{action "authenticate"}}>
+<form {{on "submit" this.authenticate}}>
   <label for="identification">Login</label>
-  <input type="text" id="identification" placeholder="Enter Login" value={{identification}} onchange={{action (mut identification) value="target.value"}}>
+  <Input @id="identification" @type="text" @placeholder="Enter Login" @value={{this.identification}} />
 
   <label for="password">Password</label>
-  <input type="password" id="password" placeholder="Enter Password" value={{password}} onchange={{action (mut password) value="target.value"}}>
+  <Input @id="password" @type="password" @placeholder="Enter Password" @value={{this.password}} />
 
   <button type="submit">Login</button>
 
```

Comment:
usages of `errorMessage` below have to be changed to `this.errorMessage`

Reply:
Already done locally, sorting the last lint error and pushing again, will ping for another review when it is ready 👍